### ISSUE TITLE
List user made posts by tags

### ISSUE THREAD

**Issue by BijayKumarPun** [thepracticaldev/dev.to#7728]
**Is your feature request related to a problem? Please describe.**
I can't list my articles/posts based on the tags I gave while writing/publishing them. If I click on any tag on my post, it takes me to a different page with all related articles, which is great, but there should be an additional option to narrow down the list to my posts only.

**Describe the solution you'd like**
A simple option to narrow down the search result to my posts only.

**Describe alternatives you've considered**
I tried searching via the search bar but that didn't work quite as desired. I had to manually go through all the articles, which becomes a major pain if the articles are many!

**Comment by rhymes** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @thepracticaldev/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md).

**Comment by rhymes** [comment]
Hi @BijayKumarPun, thanks for taking the time to raise this issue.

I'm closing this as a duplicate of https://github.com/thepracticaldev/dev.to/issues/1396 - I encourage you to add your comment there. 

Let me know I'm incorrect in my evaluation

**Comment by BijayKumarPun** [comment]
Hey @rhymes thank you for a quick response!
This issue indeed turns out to be a duplicate of #1396. Will look forward to the feature implementation.
Thanks!
